docs/source/api/express-middleware.mdx

```diff
@@ -70,7 +74,7 @@ The `expressMiddleware` function's second optional argument is an _object_ for c
 
 An optional asynchronous [`context` initialization function](../data/resolvers#the-context-argument). 
   
-The `context` function should return an object, which gets passed to every resolver that executes for a particular operation. This enables resolvers to share helpful context values, such as a database connection.
+The `context` function should return an object shared across all resolvers that are executing for a particular operation. This enables resolvers to share helpful context values, such as a database connection.
```

Comment:
Maybe "will execute" or "have executed"?

"Are executing" feels a little too time-specific (sounds like we're talking about the resolvers which happen to be executing right at this instant).

Reply:
Thinking about this sentence again, I feel like it can be streamlined even further I don't think we need to mention "executing operations" at all. How does this feel:

> The `context` function should return an object that all your server's resolvers share.

Reply:
I might tweak it to say the same thing but in your streamlined way still. What you have is perfectly fine, too!
>The context function should return an object that all your server's resolvers share during execution.

Reply:
Oooh yes, love this! I changed my mind, and added one more tweak 😂 : 

> The `context` function should return an object that all your server's resolvers share during an operation's execution.